<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="SHEFEENA" failures="2" tests="2" name="chrometest" time="14.682" errors="0" timestamp="2024-12-16T13:10:08 IST">
  <testcase classname="testScript.Add_SubCategoryTest" name="toAddSubCategory" time="0.175">
    <failure type="org.apache.poi.openxml4j.exceptions.NotOfficeXmlFileException" message="No valid entries or contents found, this is not a valid OOXML (Office Open XML) file">
      <![CDATA[org.apache.poi.openxml4j.exceptions.NotOfficeXmlFileException: No valid entries or contents found, this is not a valid OOXML (Office Open XML) file
at org.apache.poi.openxml4j.util.ZipArchiveThresholdInputStream.getNextEntry(ZipArchiveThresholdInputStream.java:156)
at org.apache.poi.openxml4j.util.ZipInputStreamZipEntrySource.<init>(ZipInputStreamZipEntrySource.java:94)
at org.apache.poi.openxml4j.opc.ZipPackage.<init>(ZipPackage.java:132)
at org.apache.poi.openxml4j.opc.OPCPackage.open(OPCPackage.java:319)
at org.apache.poi.ooxml.util.PackageHelper.open(PackageHelper.java:59)
at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:290)
at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:286)
at utilities.ExcelUtility.readStringData(ExcelUtility.java:22)
at testScript.Add_SubCategoryTest.toAddSubCategory(Add_SubCategoryTest.java:20)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
Caused by: java.util.zip.ZipException: Unexpected record signature: 0XC6217A20
at org.apache.commons.compress.archivers.zip.ZipArchiveInputStream.getNextZipEntry(ZipArchiveInputStream.java:296)
at org.apache.poi.openxml4j.util.ZipArchiveThresholdInputStream.getNextEntry(ZipArchiveThresholdInputStream.java:152)
... 23 more
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- toAddSubCategory -->
  <testcase classname="testScript.Login_Pagetest" name="verifyTheuserisAbleToLoginWithInvalidUsernameAndValidPassword" time="0.108">
    <failure type="java.io.IOException" message="Stream Closed">
      <![CDATA[java.io.IOException: Stream Closed
at java.base/java.io.FileInputStream.available0(Native Method)
at java.base/java.io.FileInputStream.available(FileInputStream.java:415)
at java.base/java.io.BufferedInputStream.read(BufferedInputStream.java:351)
at java.base/java.io.FilterInputStream.read(FilterInputStream.java:132)
at java.base/java.io.PushbackInputStream.read(PushbackInputStream.java:182)
at java.base/java.io.FilterInputStream.read(FilterInputStream.java:106)
at org.apache.commons.compress.archivers.zip.ZipArchiveInputStream.fill(ZipArchiveInputStream.java:861)
at org.apache.commons.compress.archivers.zip.ZipArchiveInputStream.readFromInflater(ZipArchiveInputStream.java:650)
at org.apache.commons.compress.archivers.zip.ZipArchiveInputStream.readDeflated(ZipArchiveInputStream.java:625)
at org.apache.commons.compress.archivers.zip.ZipArchiveInputStream.read(ZipArchiveInputStream.java:528)
at java.base/java.io.FilterInputStream.read(FilterInputStream.java:132)
at org.apache.poi.openxml4j.util.ZipArchiveThresholdInputStream.read(ZipArchiveThresholdInputStream.java:80)
at org.apache.poi.util.IOUtils.toByteArray(IOUtils.java:240)
at org.apache.poi.util.IOUtils.toByteArray(IOUtils.java:203)
at org.apache.poi.openxml4j.util.ZipArchiveFakeEntry.<init>(ZipArchiveFakeEntry.java:82)
at org.apache.poi.openxml4j.util.ZipInputStreamZipEntrySource.<init>(ZipInputStreamZipEntrySource.java:98)
at org.apache.poi.openxml4j.opc.ZipPackage.<init>(ZipPackage.java:132)
at org.apache.poi.openxml4j.opc.OPCPackage.open(OPCPackage.java:319)
at org.apache.poi.ooxml.util.PackageHelper.open(PackageHelper.java:59)
at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:290)
at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:286)
at utilities.ExcelUtility.readStringData(ExcelUtility.java:22)
at testScript.Login_Pagetest.verifyTheuserisAbleToLoginWithInvalidUsernameAndValidPassword(Login_Pagetest.java:43)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- verifyTheuserisAbleToLoginWithInvalidUsernameAndValidPassword -->
  <testcase classname="testScript.Login_Pagetest" name="verifytheUserisAbleToLoginWithInvalidCredentials" time="0.0">
    <ignored/>
  </testcase> <!-- verifytheUserisAbleToLoginWithInvalidCredentials -->
  <testcase classname="testScript.Login_Pagetest" name="verifytheUserIsAbleToLoginWithValidCredentials" time="0.0">
    <ignored/>
  </testcase> <!-- verifytheUserIsAbleToLoginWithValidCredentials -->
  <testcase classname="testScript.Login_Pagetest" name="verifyTheUserIsAbleToLoginWithValidUsernameAndInvalidPassword" time="0.0">
    <ignored/>
  </testcase> <!-- verifyTheUserIsAbleToLoginWithValidUsernameAndInvalidPassword -->
</testsuite> <!-- chrometest -->
